src/array_api_extra/_funcs.py

```diff
@@ -548,3 +559,279 @@ def sinc(x: Array, /, *, xp: ModuleType | None = None) -> Array:
         xp.asarray(xp.finfo(x.dtype).eps, dtype=x.dtype, device=x.device),
     )
     return xp.sin(y) / y
+
+
+_undef = object()
+
+
+class at:  # pylint: disable=invalid-name
```

Comment:
I'm not sure if this is what you want, but it'll make the methods that return `-> at`  instead of `-> Self` theoretically valid.

```suggestion
@typing.final
class at:  # pylint: disable=invalid-name
```

Reply:
let's punt on this for now, once we reach Python 3.11 there will be no need for `-> at`